test-tools/issuer-front-end/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Unreleased changes
 
+## 1.0.8
+
+- Add support for date-time attributes.
+
```

Comment:
The issuer-front-end has a backend. This backend needs to create a proof on the commitments on the attributes before the wallet can add a credential. This backend needs also to be updated to support `date-time` proofs.

Reply:
That is done already by updating the rust sdk.